docs/src/components/pricing/PricingTable.tsx

```diff
@@ -510,13 +510,21 @@ const rowHeaders: Record<string, React.ReactNode> = {
   'mui-x-development': <ColumnHead label="Development license" tooltip="For active development" />,
   'mui-x-updates': <ColumnHead label="Access to new releases" />,
   // Support
-  community: <ColumnHead {...{ label: 'Community' }} />,
+  community: (
+    <ColumnHead
+      {...{
+        label: 'Community support for MUI Core',
+        tooltip:
+          'The Support for MUI Core and MIT modules is provided by the community. MUI maintainers focus on solving root issues rather than offering direct support to the community at large.',
```

Comment:
```suggestion
          'Support for MUI Core and other MIT licensed code is provided by the community. MUI maintainers focus on solving root issues rather than offering direct support to the community at large.',
```